edisgo/tools/tools.py

```diff
@@ -209,6 +311,17 @@ def select_cable(edisgo_obj, level, apparent_power):
         'lv'.
     apparent_power : float
         Apparent power the cable must carry in MVA.
+    length : float
+        Length of the cable in km. Default: 0.
+    max_voltage_diff : float
+        Maximum voltage difference in pu. Default: None.
```

Comment:
Please elaborate this a bit more. It is the maximum allowed voltage difference over the line that connects the new component to the grid. If it is not specified, it is taken from the config files (section grid_connection, parameters lv_max_voltage_deviation and mv_max_voltage_deviation).